### DISCUSSION TITLE
Yum and Dnf unable to install Active Directory tools

### DISCUSSION THREAD

**Discussion by pipMinder00** [Security-Onion-Solutions/securityonion#11658] [2.4] [answered]
Hi all, 
I want to install sssd, adcli, samba-common-tools and sssd-tools 
But yum wasn't able to install. 

For sssd and adcli packages there is a conflict.

For samba-common-tools and sssd-tools the yum cannot recognize them.

Any suggestions or help for this issue? 

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/56893695/e562cc6e-69d6-47f2-b900-d905173bac76)

**Comment by TOoSmOotH** [comment, marked as answer]
We only host the packages necessary for SO. You would need to add any repos that you want to have additional packages from. Keep in mind some packages can conflict the SO so use at your own risk.
